### ISSUE TITLE
Unclear instructions

### ISSUE THREAD

**Issue by Willytor** [GiovanH/tuhc-bcplayer#1]
Not sure what the instructions mean by "music browser actions". I have enabled the mod and yet nothing seems different. I would appreciate some help

**Comment by Willytor** [comment]
address bar needs to be on

**Comment by Niklink** [comment]
I agree this is an issue, it needs be updated to either have the icon available without the address bar or explain in the mod summary that you need to enable it to see the button.
